docs-client/src/containers/App/index.tsx

```diff
@@ -477,12 +496,15 @@ class App extends React.PureComponent<Props, State> {
     return `${classes.httpMethodCommon} ${httpMethodClass}`;
   };
 
-  private fetchSpecification = async () => {
+  private initSpecification = async () => {
     const httpResponse = await fetch('specification.json');
-    const specification: SpecificationData = await httpResponse.json();
-    this.setState({
-      specification: new Specification(specification),
+    const specificationData: SpecificationData = await httpResponse.json();
+    const specification = new Specification(specificationData);
+    let openServices = {};
+    specification.getServices().forEach((service) => {
+      openServices = { ...openServices, ...{ [service.name]: true } };
```

Comment:
I think this works

```ts
openServices = { ...openServices, [service.name]: true };
```

Or also can use reduce.

```ts
const openServices = specification.getServices().reduce((services, service) => (
  { ...services, [service.name]: true }), {});
```

Reply:
Didn't know that I can do it in that way. Fixed, thank you~!